src/components/experiment-tracking/run-metadata/run-metadata.js

```diff
@@ -131,33 +136,43 @@ const RunMetadata = ({
                       {sanitiseEmptyValue(run.title)}
                     </span>
                   </td>
-                  <td className="details-metadata__table-label">Created By</td>
-                  <td className="details-metadata__table-label">
-                    Creation Date
-                  </td>
-                  <td className="details-metadata__table-label">Git SHA</td>
-                  <td className="details-metadata__table-label">Git Branch</td>
-                  <td className="details-metadata__table-label">Run Command</td>
-                  <td className="details-metadata__table-label">Notes</td>
+                  {activeTab !== 'Plots' ? (
```

Comment:
hey maybe not in this ticket but we should think about how we will separate the content for Plots and Overview tab, and maybe make them 2 separate components. As this check `activeTab !== 'Plots'` works for now but i have a feeling it will get complicated in the future when we actually make the metrics/plots etc